packages/server/test.sh

```diff
@@ -21,6 +21,13 @@ while [[ "$#" -gt 0 ]]; do
   shift
 done
 
+if ! ph_isready -h localhost -p 5432; then
```

Comment:
What does `ph_isready` do? Is this something I need to install?

Reply:
Oh...it is a typo I think...pg_isready?

Reply:
How does the script behave when pg_isready is not installed? Worth testing it?

Reply:
Fixed the typo-- pg_isready is a utility that comes along with Postgres to make sure a db is ready.  It wasn't enough to check for the listening port.  I'll add a check for the pg_isready utility.

Reply:
Good catch! This is good. Thanks!